src/main/kotlin/com/ssu/commerce/auth/controller/point/PointController.kt

```diff
@@ -23,15 +23,16 @@ class PointController(
 ) {
     @GetMapping("/history")
     fun getPointTransaction(
-        @Authenticated @Parameter(hidden = true) authInfo: AuthInfo,
+        @AuthenticationPrincipal @Parameter(hidden = true) principal: SsuCommerceAuthenticatedPrincipal,
         @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) fromDateTime: LocalDate,
         @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) toDateTime: LocalDate
-    ): List<PointTransactionProjection> = pointService.getPointTransaction(authInfo.userId, fromDateTime, toDateTime)
+    ): List<PointTransactionProjection> =
+        pointService.getPointTransaction(principal.userId, fromDateTime, toDateTime)
 
     @PostMapping("/account")
     fun createPointAccount(
-        @Authenticated @Parameter(hidden = true) authInfo: AuthInfo
-    ) = pointService.createPointAccount(authInfo.userId)
+        @AuthenticationPrincipal @Parameter(hidden = true) principal: SsuCommerceAuthenticatedPrincipal,
+    ) = pointService.createPointAccount(principal.name)
```

Comment:
As Is
@Authenticated @Parameter(hidden = true) authInfo: AuthInfo
To Be
@AuthenticationPrincipal @Parameter(hidden = true) principal: SsuCommerceAuthenticatedPrincipal,